lib/async/io/generic.rb

```diff
@@ -23,6 +23,10 @@
 
 module Async
 	module IO
+		# The default block size for IO buffers.
+		# BLOCK_SIZE = ENV.fetch('BLOCK_SIZE', 1024*16).to_i
```

Comment:
(Is this commented-out code line meant to be there?)

Reply:
Yeah, I sometimes use it for benchmarking different block sizes :p But I don't think it's suitable for release since it's a bit of a hack.